ext/intl/locale/locale_methods.c

```diff
@@ -274,27 +274,27 @@ static zend_string* get_icu_value_internal( const char* loc_name , char* tag_nam
 			}
 		}
 
-	if( fromParseLocale==1 ){
-		zend_off_t singletonPos = 0;
+		if( fromParseLocale==1 ){
```

Comment:
There's no change to this block of code other than formatting. I indented it one more tab, so that it's easier to read and understand.